_targets.R

```diff
@@ -28,6 +28,11 @@ drb_huc8s <- c("02040101","02040102","02040103","02040104","02040105","02040106"
 drb_segs_spatial <- c("31_1","135_1","233_1","236_1","244_1","249_1","332_1","354_1","355_1","358_1","396_1","593_1","1256_1","2137_1",
                       "2138_1","2757_1","2758_1","2759_1","2765_1","2766_1","2767_1","2772_1","2797_1")
 
+# Define the PRMS segments that were split within the USGS-R/delaware-model-prep pipeline; these 
+# segments require special handling to return proper catchment areas in place of NHGFv1 HRUs
+GFv1_segs_split <- c("3_1","3_2","8_1","8_2","51_1","51_2")
```

Comment:
Instead of manually defining this list of segments requiring special handling, can id them from `p2_drb_comids_all_tribs`. Something along the lines of 
`p2_drb_comids_all_tribs$PRMS_segid[grepl('_2', p2_drb_comids_all_tribs$PRMS_segid)]` where we subset to the PRMS_segid that have both `_1` and `_2`. 

I'd suggest the same for the list above, but possibly those are a bit more difficult to identify.

Reply:
I'm going to opt to leave this manually defined list for the time being since these segments are well-known to us at this point. If the underlying network changes, I agree that your suggestion would catch any `subsegseg`'s with multiple `subsegid`'s.